views/testRename.mako

```diff
@@ -16,7 +16,7 @@
     % if app.PROCESS_METHOD == 'symlink':
 <div class="text-center">
 <div class="alert alert-danger upgrade-notification hidden-print" role="alert">
-    <span>WARNING: Your current process method is SYMLINK. Renaming these files will break all symlinks in your seeding directory for this show</span>
+    <span>WARNING: Your current process method is SYMLINK. Renaming these files will break all symlinks created in Post-Processor for this show</span>
```

Comment:
@fernandog this doesn't really work either. It should state Seeding or post processor or something else maybe as this makes it look like it won't affect the seeding dir.